core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -958,6 +960,7 @@ public void year() {
     assertEquals(integerValue(2020), eval(expression));
   }
 
+  @Disabled
```

Comment:
does @DisabledOnJre works?

Reply:
How is it different from `@Disabled`? I see it takes in a value, what should I provide there?

Reply:
Added DisabledOnJre for Java 8